CreeDictionary/CreeDictionary/settings.py

```diff
@@ -188,9 +172,19 @@
         "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
     )
 
+
+class RunMainFilter(logging.Filter):
+    """
+    When DEBUG is True, django clones two processes, one is the main processes, while the other is for hot swapping.
+    The main process sets RUN_MAIN to true
+    """
+
+    def filter(self, record):
+        return os.environ.get("RUN_MAIN") == "true"
+
+
 log_dir = Path(BASE_DIR) / "django_logs"
 log_dir.mkdir(exist_ok=True)
-
 LOGGING = {
     "version": 1,
     "disable_existing_loggers": False,
```

Comment:
Logging should _not_ be written to a file in production. The recommendation in Docker is to log to `stdout`/`stderr`! [[1]][] [[2]][]. So please reconfigure this to _only_ use the stream handler in production.

[1]: https://success.docker.com/article/logging-best-practices#applicationlogs
[2]: https://www.digitalocean.com/community/tutorials/how-to-build-a-django-and-gunicorn-application-with-docker#step-5-%E2%80%94-configuring-logging